cico_setup.sh

```diff
@@ -46,7 +46,7 @@ function install_deps() {
 
 function build() {
   echo 'CICO: Cloning keycloak source code repo'
-  git clone https://github.com/fabric8-services/keycloak.git --branch master
+  git clone -b 3.2.0.Final  --depth 1 https://github.com/fabric8-services/keycloak.git
```

Comment:
Just a question, I might miss something but we're not using on purpose `master`. Why do we want to start using again different branches ? @alexeykazakov